config/routes.rb

```diff
@@ -2,8 +2,21 @@
 
 Rails.application.routes.draw do
   resources :users, only: %i[new create]
+  resource :user, only: [:update]
```

Comment:
Are we using this endpoint? If that the case please use `only: :update` as it's only one method